fetch_teleop/src/joystick_teleop.cpp

```diff
@@ -567,6 +568,188 @@ class HeadTeleop : public TeleopComponent
   boost::shared_ptr<client_t> client_;
 };
 
+class ArmTeleop : public TeleopComponent
+{
+public:
+  ArmTeleop(const std::string& name, ros::NodeHandle& nh) : 
+    init_point_(0)
+  {
+    ros::NodeHandle pnh(nh, name);
+
+    pnh.param("axis_x", axis_x_, 3);
+    pnh.param("axis_y", axis_y_, 0);
+    pnh.param("axis_z", axis_z_, 1);
+    pnh.param("axis_roll", axis_roll_, 4);
+    pnh.param("axis_pitch", axis_pitch_, 5);
+    pnh.param("axis_yaw", axis_yaw_, 2);
+
+    pnh.param("button_body_frame", button_body_frame_, 9);
+    pnh.param("button_end_effector_frame", button_end_effector_frame_, 11);
```

Comment:
This is not the same ones. I assigned different buttons for frame selection. Am I not supposed to assign more buttons?